hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/benefitInformation.jsp

```diff
@@ -24,27 +24,36 @@
 
 <div id="${n}dl-benefit-summary" class="fl-widget portlet dl-benefit-summary hrs">
   <div class="dl-banner-links">
-    <c:choose>
-    <c:when test="${enrollmentFlag == 'O'}">
+    <sec:authorize ifAnyGranted="ROLE_VIEW_NEW_HIRE_BENEFITS">
       <div class="dl-banner-link">
-        <spring:message code="benefit.summary.enrollment.O.message" text="You have a benefit enrollment opportunity. Please enroll online by clicking the following link: "/>
-        <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}"><spring:message code="open.enrollment" text="Open Enrollment" /></a>
-        <c:if test="${isMadisonUser}">
-          ${learnMoreEBenefitGuide}
-        </c:if>
+        You have a benefit enrollment opportunity.
+        <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}">Enroll now</a>
+        <c:choose>
+          <c:when test="${isMadisonUser}">
+            <a target="_blank" href="https://hr.wisc.edu/benefits/new-employee-benefits-enrollment/">Learn more</a>
+          </c:when>
+          <c:otherwise>
+            <a target="_blank" href="https://www.wisconsin.edu/ohrwd/benefits/">Learn more</a> 
+          </c:otherwise>>
```

Comment:
Stray extra `>` fixed immediately following merge directly on `uw-master` in https://github.com/UW-Madison-DoIT/hrs-portlets/commit/c5b4cc44d09be4b235971ac89d3b68b48216dcf3 .